core-bot-with-clujs/deploymentTemplates/deployUseExistResourceGroup/readme.md

```diff
@@ -0,0 +1,48 @@
+# Usage
```

Comment:
Did we write these deployment steps/templates? Or were they present in the original JS script?

Reply:
They were already present.